wp-menu-cart.php

```diff
@@ -622,7 +691,12 @@ public function wpmenucart_menu_item() {
 			$menu_item_classes .= ' ubermenu-target';
 		}
 
-		$menu_item = '<a class="'.$menu_item_classes.'" href="'.$menu_item_href.'" title="'.$menu_item_title.'">';
+		// use 'div' instead of 'a' on block editor
+		if ( $this::$is_block_editor ) {
+			$menu_item = '<div class="'.$menu_item_classes.'">';
+		} else {
+			$menu_item = '<a class="'.$menu_item_classes.'" href="'.$menu_item_href.'" title="'.$menu_item_title.'">';
+		}
```

Comment:
while this works fine for this single link, I wonder if there's a smarter way, because we will need this for Pro too (which has a lot more links in the submenu). Could we not just strip all `href` attributes from the HTML here?
https://github.com/wpovernight/wp-menu-cart/blob/e9a6d4183c71710501c6875ac391cb19eed379b4/wp-menu-cart.php#L511-L516

something like:
```php
return preg_replace('/(<[^>]+) href=".*?"/i', '$1', $menu_item);
```

the added benefit is that it maintains the `<a>` elements so that styles don't have to be adjusted